docs/index.md

```diff
@@ -73,31 +97,18 @@ That's it! You have now installed DDEV on WSL2. If you're using WSL2 for DDEV (r
 
 **Make sure you put your projects in the Linux filesystem (e.g. /home/<your_username>), not in the Windows filesystem (/mnt/c), because you'll get vastly superior performance on the Linux filesystem.**
 
-Note that nfs-mount-enabled (and running NFS) are not required on WSL2 because it's perhaps faster without NFS.
+Note that mutagen-enabled or nfs-mount-enabled (and running NFS) are not required on WSL2 because it's so fast even without these options.
+
+### Windows (traditional/legacy)
 
-### Installation or Upgrade - Windows (legacy)
+DDEV does work fine on the Windows side, although it's quite a bit slower by default, but good results have been reported by users who enabled mutagen, `ddev config global --mutagen-enabled`.
```

Comment:
`quite a bit slower then WSL2 by default`?

Reply:
Way slower, somewhat slower even with mutagen.

Reply:
I meant to add `than WSL2`. :)

Reply:
WIndows traditional is *massively* slower than WSL2. Perhaps 10x, perhaps more.

Reply:
```suggestion
DDEV does work fine on the Windows side, although it's quite a bit slower than WSL2 by default, but good results have been reported by users who enabled mutagen, `ddev config global --mutagen-enabled`.
```